composer.json

```
@@ -6,19 +6,22 @@
     "minimum-stability": "dev",
     "prefer-stable": true,
     "require": {
-        "drupal/core": "^8.9",
-        "php": ">=7.2"
+        "php": ">=7.3",
+        "drupal/core-recommended": "^8.9 || ^9 || ^10"
     },
     "require-dev": {
-        "composer/installers": "~1.9",
-        "cweagans/composer-patches": "^1.7",
-        "drupal/core-composer-scaffold": "^9.0",
-        "drupal/drupal-extension": "~4.1",
-        "drush/drush": "^10.3",
-        "openeuropa/code-review": "~1.0",
-        "openeuropa/drupal-core-require-dev": "^8.9 || ^9",
-        "openeuropa/task-runner": "~1.0.0-beta6",
-        "phpunit/phpunit": "~6 || ~7"
+        "composer/installers": "^1.11",
+        "drupal/core-composer-scaffold": "^8.9 || ^9 || ^10",
+        "drupal/drupal-extension": "~4.0",
+        "drush/drush": "^10.4",
```

```suggestion
        "drush/drush": "^10.3",
```
it is minimum supported version